README.md

```diff
@@ -0,0 +1,43 @@
+# complate-express
+
+Server-side rendered component based template-engine for express.
+
+This library is based on [complate-stream](https://github.com/complate/complate-stream).
+
+## Quick Start Guide
+
+Install complate-express to your express application
+
+```
+npm install complate-express
+```
+
+```
+const express = require('express')
+const complate = require('complate-express')
+
+const app = express();
+
+// add complate middleware
+app.use(complate('PATH_TO_COMPLATE_RENDERER_BUNDLE'))
+
+
+app.get('/', (req, res) => {
+  // after adding the middleware, `res.complate` is available for rendering
+  res.complate('my-component', { config })
+})
+```
+
+## API
+
+#### complate(rendererPath: String|Renderer, options: Object)
```

Comment:
you might want to mention that we're not using Express's view-engine mechanism because it doesn't support streaming - cf. http://markojs.com/docs/express/#skip-the-view-engine